### DISCUSSION TITLE
DB_566128M5MP_W.2602_23081801.Reolink-Video-Doorbell-WiFi.OV05A10.5MP.WIFI8812.REOLINK

### DISCUSSION THREAD

**Discussion by hapklaar** [AT0myks/reolink-fw-archive#52] [Firmware sharing]
Provided by Reolink support for much improved RTSP performance. Unfortunately the newer 3215 firmware has regressed in this matter and is a much worse experience, giving lower framerate and drops.

FIle is too big to upload here

**Comment by AT0myks** [comment]
Could you provide the link that they sent you?

**Reply by hapklaar** [reply]
Sure, here you go

https://support-d.reolink.com/attachments/token/WXHcHREOjF2vQIsz5nlUxh2Jh/?name=[DB_566128M5MP_W.2602_23081801.Reolink-Video-Doorbell-WiFi.OV05A10.5MP.WIFI8812.REOLINK.pak](https://support-d.reolink.com/attachments/token/WXHcHREOjF2vQIsz5nlUxh2Jh/?name=DB_566128M5MP_W.2602_23081801.Reolink-Video-Doorbell-WiFi.OV05A10.5MP.WIFI8812.REOLINK.pak
